.circleci/config.yml

```diff
@@ -205,6 +216,24 @@ path_to_live: &path_to_live
         unit_test: false
         filters: { branches: { only: [master] } }
 
+    - build_containers_and_push_to_ecr/docker_build_lambda_image:
+        name: perfplat-lambda-docker-build-preproduction
+        account_id: "987830934591"
```

Comment:
Should this be a configuration variable?

Reply:
ideally, our counterparts on the Use team also do this in their CI config however. might be less of an issue on the migration to Github Actions.